### DISCUSSION TITLE
zh/config/style

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#476] [Announcements]
# zh/config/style

你可以在 .vuepress/styles 中通过在 config.scss 和 palette.scss 文件中写入变量值来更改主题的样式。 你也可以在 .vuepress/styles/index.scss 中添加你自己的样式。

https://theme-hope.vuejs.press/zh/config/style.html

**Comment by rockbenben** [comment]
默认值均显示为 `File not found`

**Reply by Mister-Hope** [reply]
感谢指出，我们会在下个版本文档中修复
